nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -975,141 +949,82 @@ define([
         }
 
         function buildRunControlPanel(events) {
-            return div({ dataElement: 'run-control-panel' }, [
-                div(
-                    {
-                        style: {
-                            border: '1px silver solid',
-                            height: '50px',
-                            position: 'relative',
-                            display: 'flex',
-                            flexDirection: 'row',
+            const cssBaseClass = 'kb-rcp';
```

Comment:
All inline styles have been removed and css info is now in the SCSS file. I also removed some extra `div`s at the same time.